ci/test-conformance-eks.sh

```diff
@@ -190,7 +192,11 @@ function deliver_antrea_to_eks() {
 function run_conformance() {
     echo "=== Running Antrea Conformance and Network Policy Tests ==="
 
-    ./run-k8s-e2e-tests.sh --e2e-conformance --e2e-network-policy > eks-test.log
+    if [[ -z ${GIT_CHECKOUT_DIR+x} ]]; then
+        GIT_CHECKOUT_DIR=..
+    fi
+    skip_regex="\[Slow\]|\[Serial\]|\[Disruptive\]|\[Flaky\]|\[Feature:.+\]|\[sig-cli\]|\[sig-storage\]|\[sig-auth\]|\[sig-api-machinery\]|\[sig-apps\]|\[sig-node\]|NodePort"
```

Comment:
one more thing: does adding "NodePort" to the skip regex only remove the 2 tests which are failing? or does it exclude additional tests as well?

Reply:
That's a good catch. Found out that for 1.16, excluding 'NodePort' will skip 3 test:
- [sig-network] Services should be able to change the type from ExternalName
    to NodePort [Conformance]
-[sig-network] Services should be able to change the type from NodePort
    to ExternalName [Conformance]
- [sig-network] Services should be able to create a functioning NodePort
    service [Conformance]
The reason why the second test would have succeeded before this fix is that in the actual testing it simply verified the nodeport service is created without error, but not traffic: https://github.com/kubernetes/kubernetes/blob/f13051c28c69e0d44dfe4c0e2989cd5fec47b7d2/test/e2e/network/service.go#L1718
Since we're skipping the nodeport testcases I'm assuming this can as well be tested with 
'[sig-network] Services should be able to change the type from ClusterIP
    to ExternalName [Conformance]'?

Reply:
yes sounds good, let's skip all the NodePort tests to avoid future issues (if the test is updated for example).

could you add a comment to the script explaining why we are skipping the NodePort tests? A link to the issue (https://github.com/vmware-tanzu/antrea/issues/690) could suffice.

Reply:
Done